MartinP460/react-denmark-map
# feat: This is part 2 of changing the folder structure and in this iss…

### Changes

This is part 2 of changing the folder structure and in this is the changes should be made in the demo app.

### How to test

1) CD into packages/core && `npm run build`
2) CD into demo folder
3) `npm run dev`
4) Check out that the tabs still work in the `demo` section
